backend/image-upload-api/flask-api/main.py

```diff
@@ -1,42 +1,78 @@
-
+# Imports ------------------------------------------------------------------------------------------------------------
 from app import app
 from flask import Flask, request, redirect, jsonify, send_from_directory
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+import json
+from PIL import Image,  ImageOps
+from tensorflow.keras.models import model_from_json
 
-
-# Allowed file types
+# Allowed file types for the 
 ALLOWED_EXTENSIONS = set(['jpeg', 'png', 'jpg'])
-
-
 # File extension must be from ALLOWED_EXTENSIONS
 def allowed_file(filename):
     return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
 
 
-# serving index.html on flask
+# Deserialize the json file and create the model-----------------------------------------------------------------------
+with open("./notebooks/models/2021-09-06-keras.json", "r") as json_file:
+    json_model = json.load(json_file)
+
+model_test = model_from_json(json_model)
+
+# Some functions for resp ---------------------------------------------------------------------------------------------
+def result(array):
+    if array[0][0] > array[0][1]:
+        return 'genuine'
+    elif array[0][0] < array[0][1]:
+        return 'forged'
+    else:
+        return 'unable to determine'
+def result_percentage(array):
```

Comment:
Add a docstring here too and make sure there's a blank line in between the methods